core/sqf/sqenvcom.sh

```diff
@@ -284,6 +297,8 @@ elif [[ -d /opt/cloudera/parcels/CDH ]]; then
   export CURL_INC_DIR=/usr/include
   export CURL_LIB_DIR=/usr/lib64
 
+  lv_hbase_cp=`hbase classpath`
```

Comment:
This will need to be set for Cloudera packages install as well. 

Add lv_hbase_cp=`hbase classpath` after elif [[ -n "$(ls /usr/lib/hadoop/hadoop-*cdh*.jar 2>/dev/null)" ]]; then
(around line 340).